examples/src/main/scala/com/criteo/cuttle/examples/HelloTimeSeriesWithManyJobs.scala

```diff
@@ -0,0 +1,40 @@
+// Example: Hello cuttle timeseries with load!
```

Comment:
This is really useful, but should not be part of "examples" IMO. Maybe move it to tests ?

Reply:
Done, had to tweak `build.sbt` for using loop on test, also tweaked the test for having long ids / hundreds of jobs tags for other tests